lib/middleware/resource.js

```diff
@@ -8,7 +8,7 @@ function factory ({ loader }) {
       resources = await loader.forPropertyOperation(req.hydra.term)
     }
 
-    if (resources.length !== 1) {
```

Comment:
What's the intention of changing this? Later resources is accessed via `[0]`, so if there are more than one a random one would be used.

Reply:
That does not change, does it?
The problem currently is `length === 0`

Reply:
OK, it will run into the else branch and do nothing. Let's clean this up later and add a more explicit code branch for no resource found when we add the separate resource choosing logic.